fix(py,ts): support OME-Zarr RFC-3 (arbitrary axis dimensions, names, types, ordering) (#611)
thewtex/ngff-zarr: docs/spec_features.md

- **OME-Zarr v0.4 to v0.6**: Writes OME-Zarr versions 0.4 to 0.6, including
  RFC-4 anatomical orientation. v0.6 additionally adds RFC-5 coordinate systems
  and transformations.
- **OME-Zarr 0.9.dev1**: Reads and writes the development version that adopts

>   RFC-3, which lifts every restriction on the number, names, types and order

Reviewer: ```suggestion
  RFC-3, which extends support for the number, names, types and order
```
